src/components/display/ListV2.test.tsx

```diff
@@ -67,4 +67,30 @@ describe('List', () => {
 		expect(items[0].onClick).toHaveBeenCalled();
 		expect(items[1].onClick).not.toHaveBeenCalled();
 	});
+
+	it('should not invoke onVisible when element is already visible and the callback reference change', async () => {
+		const items = [
+			{
+				id: '1',
+				name: 'item 1'
+			},
+			{
+				id: '2',
+				name: 'item 2'
+			}
+		];
+
+		const listItems = items.map((item) => (
+			<ListItem key={item.id}>{(): React.JSX.Element => <div>{item.name}</div>}</ListItem>
+		));
+		const onListBottomFn1 = jest.fn();
+		const onListBottomFn2 = jest.fn();
+
+		const { rerender } = setup(<ListV2 onListBottom={onListBottomFn1}>{listItems}</ListV2>);
+		triggerLoadMore();
+		expect(onListBottomFn1).toBeCalledTimes(1);
```

Comment:
isn't toBeCalled deprecated in favor of toHaveBeenCalled? :stuck_out_tongue_winking_eye:

Reply:
It seems you are right! bEaSLint :)